### PR TITLE
Data archiving workflow instructions editing

### PR DESCRIPTION
**Pull request by ottumm**
- link to data archiving instructions from main README
- minor copy-editing

### CODE REVIEW

**Comment by ottumm** [comment]
Open question: should I add instructions around finding a checker (via e.g. the #epa channel) and doing the checking to the bottom of `harvesting.md`, or will the pipeline app handle that flow?

**Comment by ottumm** [comment]
Spoke with @suchthis and it sounds like we're not doing any checking at the event, which simplifies everything nicely!

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@
   * Join the #data-archiving slack channel
   * DM a guide your email address for an invite to the pipeline app
   * Please use the same username for both slack and app ("firstnamelastname" or whatever you are comfortable with having online)
+  * Follow the [data archiving instructions](./workflow/harvesting.md)
```
Comment (suchthis) [inline README.md]: Did you want to link specifically to the harvesting workflow here? Seems like people in this track need to follow both/either the research workflow and the harvesting workflow, and the research comes first.

Comment (ottumm) [reply README.md]: Ah good call.

File: workflow/harvesting.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
-# Harvesting
+# Data Archiving (aka "Harvesting")
```
Comment (suchthis) [inline workflow/harvesting.md]: Do you think we should stick with using "data archiving" to mean the track broadly (research & harvest) or should we align it specifically with the harvesting part?

Comment (ottumm) [reply workflow/harvesting.md]: Yep, you're right. "Data Archiving" is the name of the track and "Harvesting" is the name of the activity. I'll change it back.

**Pull request merged**